### PR TITLE
Update EN Doc: Create Pipeline Using a Jenkinsfile

### PR DESCRIPTION
**Pull request by Felixnoo**
Signed-off-by: Felixnoo <felixliu@yunify.com>

### CODE REVIEW

**Comment by Felixnoo** [comment]
Need to avoid conflicts with #1296.

File: content/en/docs/devops-user-guide/how-to-use/create-a-pipeline-using-jenkinsfile.md [hunk 1]
```diff
@@ -65,9 +65,13 @@ There are eight stages as shown below in this example pipeline.
 
    ![sonar-token](/images/docs/devops-user-guide/using-devops/create-a-pipeline-using-a-jenkinsfile/sonar-token.jpg)
 
-3. In total, you have four credentials in the list.
+3. You also need to create a GitHub personal access token with the permission as shown in the below image, and then use the generated token to create another Account Credentials, for example, `github-token`, in your DevOps project. 
 
-   ![credential-list](/images/docs/devops-user-guide/using-devops/create-a-pipeline-using-a-jenkinsfile/credential-list.jpg)
+   ![github-token-scope](/images/docs/devops-user-guide/using-devops/create-a-pipeline-using-a-jenkinsfile/github-token-scope.png)
+
```
Comment (Sherlock113) [inline content/en/docs/devops-user-guide/how-to-use/create-a-pipeline-using-jenkinsfile.md:71]: Add a note here:

To create a GitHub personal access token, go to **Settings** of your GitHub account, click **Developer settings**, select **Personal access tokens**, and click **Generate new token**.

File: content/en/docs/devops-user-guide/how-to-use/create-a-pipeline-using-jenkinsfile.md [hunk 2]
```diff
@@ -65,9 +65,13 @@ There are eight stages as shown below in this example pipeline.
 
    ![sonar-token](/images/docs/devops-user-guide/using-devops/create-a-pipeline-using-a-jenkinsfile/sonar-token.jpg)
 
-3. In total, you have four credentials in the list.
+3. You also need to create a GitHub personal access token with the permission as shown in the below image, and then use the generated token to create another Account Credentials, for example, `github-token`, in your DevOps project.
```
Comment (Sherlock113) [inline content/en/docs/devops-user-guide/how-to-use/create-a-pipeline-using-jenkinsfile.md:68]: and then use the generated token to create another Account Credentials, for example, `github-token`, in your DevOps project.
->
and then use the generated token to create Account Credentials (for example, `github-token`) for GitHub authentication in your DevOps project.

**Pull request merged**